src/main/java/com/spring/taskmanger/controller/UserController.java

```diff
@@ -24,36 +28,37 @@
 	@Autowired
 	UserService userService;
 
-	@GetMapping("/users")
-	public List<User> getAllUsers() {
-
-		return userService.getAllUsers();
-	}
+	@Autowired
+	TokenRepository tokenRepository;
 
-	@GetMapping("/users/{id}")
+	@GetMapping("/users")
```

Comment:
I think it should be "/user"? this API to get the current user, right?

Reply:
@GetMapping("/users/{id}") : is removed
it just two endpoints in UserController , 
@GetMapping("/users") for getting the current user 
@PutMapping("/users") for update the information for the current user

Reply:
Okay